src/components/study-container.js

```diff
@@ -265,68 +122,15 @@ export function StudyContainer({ view, onChangeTab }) {
 
     const currentNodeRef = useRef();
 
-    const [loadFlowStatus] = useNodeData(
-        studyUuid,
-        currentNode?.id,
-        fetchLoadFlowStatus,
-        loadFlowStatusInvalidations,
-        RunningStatus.IDLE,
-        getLoadFlowRunningStatus
-    );
-
-    const [securityAnalysisStatus] = useNodeData(
-        studyUuid,
-        currentNode?.id,
-        fetchSecurityAnalysisStatus,
-        securityAnalysisStatusInvalidations,
-        RunningStatus.IDLE,
-        getSecurityAnalysisRunningStatus
-    );
-
-    const [sensiStatus] = useNodeData(
-        studyUuid,
-        currentNode?.id,
-        fetchSensitivityAnalysisStatus,
-        sensiStatusInvalidations,
-        RunningStatus.IDLE,
-        getSensiRunningStatus
-    );
-
-    const [shortCircuitStatus] = useNodeData(
-        studyUuid,
-        currentNode?.id,
-        fetchShortCircuitAnalysisStatus,
-        shortCircuitStatusInvalidations,
-        RunningStatus.IDLE,
-        getShortCircuitRunningStatus
-    );
-
-    const [dynamicSimulationStatus] = useNodeData(
-        studyUuid,
-        currentNode?.id,
-        fetchDynamicSimulationStatus,
-        dynamicSimulationStatusInvalidations,
-        RunningStatus.IDLE,
-        getDynamicSimulationRunningStatus
-    );
-
-    const [voltageInitStatus] = useNodeData(
-        studyUuid,
-        currentNode?.id,
-        fetchVoltageInitStatus,
-        voltageInitStatusInvalidations,
-        RunningStatus.IDLE,
-        getVoltageInitRunningStatus
-    );
+    // fetches run button status and keep them updated according to notifications
```

Comment:
Technically we fetch the status of the computing / analysis, not the status of the button

Reply:
fixed